src/cloudant/account.py

```diff
@@ -178,17 +160,18 @@ def all_dbs(self):
 
     def create_database(self, dbname, **kwargs):
         """
-        _create_database_
+        Creates a new database on the remote server with the name provided.
+        The method will also add the new database object to the client's
```

Comment:
Maybe this could start as 'this method will...', or possibly combine the two sentences together, something like 'Creates a new database with the name and adds the new database object...' ?